store/sqlstore/store.go

```diff
@@ -321,10 +326,17 @@ func (ss *SqlStore) GetCurrentSchemaVersion() string {
 	return version
 }
 
-func (ss *SqlStore) GetDbVersion() (string, error) {
+// GetDbVersion returns the version of the database being used.
+// If numerical is set to true, it attempts to return a numerical version string
+// that can be parsed by callers.
+func (ss *SqlStore) GetDbVersion(numerical bool) (string, error) {
 	var sqlVersion string
 	if ss.DriverName() == model.DATABASE_DRIVER_POSTGRES {
-		sqlVersion = `SHOW server_version`
+		if numerical {
+			sqlVersion = `SHOW server_version_num`
+		} else {
+			sqlVersion = `SHOW server_version`
+		}
```

Comment:
I am wondering if a separate method that directly returns an `int` would be more appropriate. Alternatively, we could return three values `(string, int, error)` without the need for a parameter. Maybe we could use the same convention (2 digits per version) for other db types and provide a conversion accordingly? (this could be done as a separate, even help-wanted ticket).

Reply:
I had thought about this one. I didn't go for a separate method because this was just for postgres. But now I checked MySQL and it directly returns a single version string for `select version()`. I am not terribly against doing this in this PR itself, but an HW ticket for a separate method along with the parsing and converting to int should also be okay. Since this is not called anywhere else in the codebase except for telemetry.

Let me know.

Reply:
Yes, I like the HW route.